### PR TITLE
By fingerprint

### PR DESCRIPTION
**Pull request by zariye**
=)

### CODE REVIEW

File: test/integration/nickserver_test.rb [hunk 1]
```diff
@@ -40,15 +40,13 @@ def test_GET_served_via_SKS
     end
   end
 
-  def test_POST_served_via_SKS
```
Comment (elijh) [inline test/integration/nickserver_test.rb:43]: why remove test for requests via POST? I like supporting either GET or POST, depending on the preference of the client.

**Comment by zariye** [comment]
@azul and me felt like it is a bit redundant, since it tests the same code path and the client uses just GET so far. But we don't really mind to add them...do you think it is important for future implementation?

**Comment by elijh** [comment]
I thought I had a good reason for supporting both POST and GET, but I can't think of anything now, so maybe there is no good reason. In general, I like it when APIs allow you to use either get or post. my assumption has always been that people will use get when testing and post will be used by client code, but I guess that is not the case. GETs are potentially more awkward in terms of how you have to encode the request, but i guess these days the client http libraries take care of that for you. If nickserver is behind apache, then apache by default will be logging the get params but not the post params, but we can take care of that in apache. Or, with better tls support now in nickserver, get rid of the apache proxy sitting in front of nickserver.

So, I guess I am OK with getting rid of POST, if for no other reason than my arguments in favor of keeping it are flimsy.

In the future, we might have an issue with GET-only if/when we support the ability of clients to submit a list of fingerprints to check for key updating. I don't know how large these lists will get, but it is generally considered good to keep GETs under 2k, although the limit is entirely dependent on the client/server.

**Comment by azul** [comment]
@elijh We're not checking the http verb anywhere and we still parse the request body for params. So nothing changed about the functionality. We're just not integration testing the code path with both options. I don't think the additional test adds much confidence in the codebase but it adds quite a bit of redundancy. I'm fine with continueing testing the POST - but then i think i will refactor the test.

**Pull request merged**